### PR TITLE
remove outdated banner from kfserving

### PR DESCRIPTION
**Pull request by animeshsingh**

### CODE REVIEW

File: content/en/docs/components/serving/kfserving.md [hunk 1]
```diff
@@ -26,7 +22,9 @@ Our strong community contributions help KFServing to grow. We have a Technical S
 
 ## Install with Kubeflow
 KFServing works with Kubeflow 1.1. Kustomize installation files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
-See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) and [GCP](https://github.com/kubeflow/kfserving/tree/master/docs/samples/gcp-iap)
+See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs. 
+
+Kubeflow 1.1 includes KFServing v0.3, where the focus has been on providing more stability by doing a major move to KNative v1 APIs. Additionally, we added GPU support for PyTorch model servers, and pickled model format support for SKLearn. There were other enhancements vis a vis routing, payload logging, bug fixes etc., details of which can be found [here](https://github.com/kubeflow/kfserving/releases/tag/v0.3.0).
```
Comment (yuzisun) [inline content/en/docs/components/serving/kfserving.md:27]: is `vis a vis` a french word ?

File: content/en/docs/components/serving/kfserving.md [hunk 2]
```diff
@@ -26,7 +22,9 @@ Our strong community contributions help KFServing to grow. We have a Technical S
 
 ## Install with Kubeflow
 KFServing works with Kubeflow 1.1. Kustomize installation files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
-See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) and [GCP](https://github.com/kubeflow/kfserving/tree/master/docs/samples/gcp-iap)
+See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs.
```
Comment (yuzisun) [inline content/en/docs/components/serving/kfserving.md:25]: are there kfserving specific docs from the cloud providers?

Comment (animeshsingh) [reply content/en/docs/components/serving/kfserving.md:25]: not kfserving specific - but when you deploy with Kubeflow.

File: content/en/docs/components/serving/kfserving.md (hunk 1 above)
Comment (animeshsingh) [reply content/en/docs/components/serving/kfserving.md]: changed the language to more formal.

**Comment by animeshsingh** [comment]
@yuzisun we need this in today

**Pull request merged**